docs/SKUSelector.md

```diff
@@ -49,6 +49,7 @@ The `sku-selector` block is mainly used in Product Details Pages (PDPs) to displ
 | `disableUnavailableSelectOptions`     | `boolean`   | Whether a product variation that leads to an impossible product combination should be displayed with the disabled attribute when `displayMode` is set to `select`. By default, all select options are without the disabled attribute set based on the type of variation | `false`  |
 | `displayMode` | `enum` | Defines how the product variation names will be displayed (it doesn't apply to product variation images). Possible values are: `default` (displays all variation names), `select` (only displays the selected variation name) or `slider` (displays all variation names in a slider when the number of available options in greater than the value defined in the `sliderDisplayThreshold` prop). Notice that this prop is _responsive_, so you can declare an object as its value specifying a value for each breakpoint (`desktop` and `mobile`). | `default`  |
 | `hideImpossibleCombinations`     | `boolean`   | Defines if a product variation should be clickable (`true`) or not (`false`). If `true`, the variation won't be displayed. If `false`, the variation is displayed with less opacity. For example, there are two colors for a pair of sneakers: pink and white, and if the pink sneaker is available for only one size, the other sizes won't be displayed (`true`).| `true`  |
+| `isSort`     | `boolean`   | Defines if the variations are to be sorted in alphabetical order.| `false`  |
```

Comment:
What do you think of a more explicitly name like `sortVariationsByLabel` or `sortByLabel`? I think it will make the code clearer